app/Repositories/CurrikiGo/Outcome/OutcomeRepository.php

```diff
@@ -47,23 +46,24 @@ private function normalizeRow($data) {
         if (strpos($data['library'], 'H5P.PersonalityQuiz') !== false) {
             $answers = [];
 
-            foreach ($data['answer'] as $i => $answer) {
-                if ($i === 0) {
-                    $res = [
-                        (is_array($answer['response'])) ? 'Quiz Result: '.$answer['response'][0] : 'Quiz Result: '.$answer['response']
-                    ];
-                } else {
-                    $res = [
-                        (is_array($answer['response'])) ? 'Response '.$i.': '.$answer['response'][0] : 'Response '.$i.': '.$answer['response']
+            if (isset($data['answer'])) {
+                foreach ($data['answer'] as $i => $answer) {
+                    if ($i === 0) {
+                        $res = [
+                            (is_array($answer['response'])) ? 'Quiz Result: '.$answer['response'][0] : 'Quiz Result: '.$answer['response']
```

Comment:
Add spaces before and after concatenation operator, e.g. 'Quiz Result: ' . $answer['response'][0] instead of 'Quiz Result: '.$answer['response'][0].